process/generic.go

```diff
@@ -228,6 +231,9 @@ func (g *Generic) Run(ctx context.Context) error {
 			return server.Run(ctx)
 		})
 	}
+	if ready != nil {
+		ready()
```

Comment:
Unlike for controllers above (the second `for g.Controllers`), this doesn't wait for server(s) to signal readiness, but it should.